css/style.css

```
@@ -0,0 +1,333 @@
+/* fonts */
+@font-face {
+  font-family: 'PT Sans';
+  src: url("../fonts/PTSans.woff2") format("woff2"),
+      url("../fonts/PTSans.woff") format("woff"),
+  font-weight: 400;
+  font-style: normal;
+}
+@font-face {
+  font-family: 'PT Sans';
+  src: url("../fonts/PTSans-Bold.woff2") format("woff2"),
+      url("../fonts/PTSans-Bold.woff") format("woff"),
+  font-weight: 700;
+  font-style: normal;
+}
+/* Variables */
+:root{
+  --basic-black:#000000;
+  --basic-dark:#333333;
+  --basic-white:#ffffff;
+  --basic-white-focus:rgba(255, 255, 255, .3);
+  --basic-special: #666666;
+  --area-special:#F2F2F2;
+  --area-grey: #EEEEEE;
+  --area-grey-focus: #BDBBBC;
+   --area-grey-line: #E5E5E5;;
+  --area-blue:#81B3D2;
+  --area-blue-hover:#669EC0;
+  --area-blue-focus:#5496BD;
+  --area-brown: #766357;
+  --area-brown-hover: #604E43;
+  --area-brown-focus: #503E33;
+}
+
+/*Global*/
+.page{
+  background-color: var(--area-grey);
+}
+.page-body{
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 0;
+  font-family: 'PT Sans' , Arial, sans-serif;
+  font-size: 14px;
+  line-height: 21px;
+  font-weight: 400;
+  text-transform: uppercase;
+  color: var(--basic-dark);
+  background-color: var(--basic-white);
+}
+a{
+  text-decoration: none;
+}
+img{
+  max-width: 100%;
+  height: auto;
+}
+ul{
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+h1, h2, h3{
+  margin: 0;
+  color: var(--basic-black);
+  font-weight: 700;
+  line-height: 26px;
+  font-size: 21px;
+}
+p{
+  margin: 0;
+}
+fieldset{
+  margin: 0;
+  padding: 0;
+  border:none;
+}
+button:focus{
+  outline: none;
+}
+.container{
+  max-width: 1056px;
+  margin: 0 auto;
+}
+.main-navigation-list a{
+  line-height: 26px;
+  color: var(--basic-black);
+  font-weight: 700;
+}
+.main-content-title{
+  margin-bottom: 16px;
+  text-align: center;
+}
+.main-content-desc{
+  text-align: center;
+  margin-bottom: 50px;
+  font-size: 14px;
+  color: var(--basic-dark);
+  font-weight: 400;
+}
+.features-item{
+  line-height: 21px;
+}
+.features-title{
+  margin:0 0 25px 0;
+}
+.features-item:first-child .features-title, .features-item:nth-child(2) .features-title, .features-item:first-child .features-text, .features-item:nth-child(2) .features-text{
```

селекторы через запятую всегда разделяй переносом строки, так намного читабельней:
```suggestion
.features-item:first-child .features-title,
.features-item:nth-child(2) .features-title,
.features-item:first-child .features-text,
.features-item:nth-child(2) .features-text{
```